### ISSUE TITLE
Set lookup field to null 

### ISSUE THREAD

**Issue by lamimad** [seanmcne/Microsoft.Xrm.Data.PowerShell#232]
How can we Set lookup field to null for a record?

The following code is not working:

```
$ImportedTheme.logoid = $null
$ImportedTheme.logoid_Property = $null
Set-CrmRecord -conn $connection -CrmRecord $ImportedTheme
```
It's about the logo of Themes

The lookup field value remains non-empty

**Comment by seanmcne** [comment]
@kenakamu are you able to look into why this isn't working?  I suspect this would apply to any lookup value - I'd expect setting to null to function properly. 

Set-CrmRecord -EntityLogicalName theme -Id $themeId -Fields @{"logoid"=$null}

Results in exception: 
MapFieldTypeByFieldValue : Cannot bind argument to parameter 'Value' because it is null.

**Comment by seanmcne** [comment]
at first glance I'm thinking it's due to "$field.Value" when $Field is null but it looks like if the value is null we should just set it to a raw field type with a value of $null vs. calling into MapFieldTypeByFieldValue

![image](https://user-images.githubusercontent.com/2292260/28733038-ec8a012c-739f-11e7-9c49-4a6c5b3694ce.png)

**Comment by kenakamu** [comment]
I think if we respect null then all the value has null value will be updated, which we don't want. They may need to use different function in that case?

**Comment by kenakamu** [comment]
okay figured it out. I will put the fix into 2.6
